bindings/protocols/modbus/ontology.ttl

```diff
@@ -129,54 +129,54 @@ binding:assignment rdf:type owl:AnnotationProperty .
 #################################################################
 
 ###  https://www.w3.org/2019/wot/modbus#hasAddress
-:hasAddress rdf:type owl:DatatypeProperty ;
+:address rdf:type owl:DatatypeProperty ;

```

Comment:
my expectation would be that hasAddress, hasCodeNumber, etc are correct. In other WoT ontologies, we use the same pattern (e.g., https://www.w3.org/2019/wot/hypermedia#hasTarget).

The context file is the place, which should do the mapping from address to hasAddress, code to hasCode, etc and vice versa.